app/Http/Controllers/TaskController.php

```diff
@@ -15,17 +15,10 @@ public function index(){
     }
     
     public function store(Request $req){
-        // $drt = date('A', strtotime($req->time));
-         $date = $req->date;
-        $time = $req->time . ' ' .$req->drt;
        
         $arr = [
             'name' => 'required | min:5 | max:20',
-            'date' => Rule::unique('tasks')->where(function ($query) use ($date, $time) {
-                return $query->where('date', $date)
-                    ->where('time', $time);
-            }),
-               'time' => 'required',
+            'date' =>  'required',
         ];
 
         $valid = Validator::make($req->all(), $arr);
```

Comment:
Why are you not using the standard way of validation as in https://laravel.com/docs/9.x/validation#quick-writing-the-validation-logic